packages/webdriverio/src/commands/element/getLocation.ts

```diff
@@ -36,7 +36,7 @@ export function getLocation (this: WebdriverIO.Element, prop: keyof Location): P
 export async function getLocation (
     this: WebdriverIO.Element,
     prop?: keyof Location
-): Promise<Location | number> {
+): Promise<any> {
```

Comment:
I don't see the need to make this `any`, what is the reasoning for this change?

```suggestion
): Promise<Location | number> {
```

Reply:
if here we use return type as `Location | number` then the declaration refrence is not working as expected for `getLocation()` it will ask for an 1 argument in the warning because reference for IDE always point to `export function getSize (this: WebdriverIO.Element, prop: keyof RectReturn): Promise<number>` declaration.

Reply:
I am not sure if this is the right solution then. We shouldn't return `any` as we loose type information about the return value. I might be wrong here though, do you have any reference documentation that recommends this approach?

Reply:
Yup you are correct that is not right, I have updated the code with a different solution